README.md

```diff
@@ -1,35 +1,36 @@
 # Shortlinks Site
 
-Small PHP script redirect user to a URL based on a keyword mach found in a CSV sheet found on the internet.
+A small PHP script that redirects to a URL based on a keyword matched against a CSV sheet on a URL.
 
+## Installation
```

Comment:
@ASoTNetworks if there is a need to set up crontab you need to document that here. Someone reading this doc should be able to deploy this fully without your help.

Reply:
I think cronjob is overkill, and also limits where it can be deployed